src/heretic/config.py

```diff
@@ -28,6 +28,11 @@ class Settings(BaseSettings):
         description="If this model ID or path is set, then instead of abliterating the main model, evaluate this model relative to the main model.",
     )
 
+    mode: Literal["remove_inhibitions", "increase_inhibitions"] = Field(
```

Comment:
I think it's cleaner and simpler to just have a boolean called `reverse` here (default `False`), given that there are only two possible directions. This saves us from using literals everywhere, and also keeps a more neutral language as not every good/bad dataset is about "inhibitions".

Reply:
shouldn't be a problem, can fix